tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "paths": {
       "@fonts/*": ["/assets/fonts/*"],
       "@styles/*": ["/styles/*"],
-      "@components": ["/components"]
+      "@components": ["/components"],
+      "@images/*": ["/assets/images/*"]
     },
   },
   "include": ["./src"],
```

Comment:
same as above, other branch for fix config...

Reply:
This will be resolved in conflict, no worries :)